websites/views/blogs_summary.view.lkml

```diff
@@ -1,5 +0,0 @@
-include: "//looker-hub/websites/views/blogs_daily_summary.view.lkml"
-
-view: blogs_summary {
-  extends: [blogs_daily_summary]
-}
```

Comment:
Does this data not generally have any metrics?

Reply:
Added some measures @ANich, would be amazing if you could re-review the PR.